packages/paste-core/components/switch/stories/index.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import {useUID} from '@twilio-paste/uid-library';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {Switch, SwitchContainer} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Switch',
+  component: Switch,
+};
+
+export const Default: React.FC = () => {
+  const id = useUID();
+  return (
+    <SwitchContainer id={id} helpText="This is help text" label="This is the label">
+      <Switch />
+    </SwitchContainer>
+  );
+};
+
+export const SwitchOnly: React.FC = () => {
+  const id = useUID();
+  return <Switch id={id} />;
+};
+
+export const LabelOnly: React.FC = () => {
+  const id = useUID();
+  return (
+    <SwitchContainer id={id} label="This is the label">
+      <Switch />
+    </SwitchContainer>
+  );
+};
+
+export const Required: React.FC = () => {
+  const id = useUID();
+  return (
+    <SwitchContainer id={id} helpText="This is help text" label="This is the label" required>
+      <Switch />
+    </SwitchContainer>
+  );
+};
+
+export const Disabled: React.FC = () => {
+  const id = useUID();
+  return (
+    <SwitchContainer id={id} helpText="This is help text" label="This is the label" disabled>
+      <Switch />
+    </SwitchContainer>
+  );
+};
+
+export const On: React.FC<{element?: string}> = ({element}) => {
+  const id = useUID();
+  return (
+    <SwitchContainer id={id} element={element} helpText="This is help text" label="This is the label">
+      <Switch on />
+    </SwitchContainer>
+  );
+};
```

Comment:
suggestion: it would be helpful here to create a story with a controlled state where clicking toggles the Switch on/off for future development and maintenance.

[edit] And for our docs examples too